src/containers/catalog/CatalogComponent.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import { EntitySetList } from './components/EntitySetList';
+import AuthService from '../../utils/AuthService';
+
+
+export class CatalogComponent extends React.Component {
+  static propTypes = {
```

Comment:
I don't have a strong opinion about one over the other. I followed the pattern in the current code, but they should transpile to the same thing. 

Typescript lets you define an interface for `props`, so you don't use `propTypes` at all. If Flow has something similar, I'd love to use that :)